bizon/connectors/sources/kafka/src/source.py

```
@@ -379,4 +380,9 @@ def get(self, pagination: dict = None) -> SourceIteration:
 
     def commit(self):
```

We should propagate the error upstream where the `commit()` function is called and log an error there so it's more explicit 👍